projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.ts

```diff
@@ -18,74 +31,85 @@ export class RangeSliderComponent implements OnInit, AfterViewInit {
   @ViewChild('maxSlider')
   public maxSlider: ElementRef;
 
-  @Output()
-  public sliderWidth: EventEmitter<any> = new EventEmitter<any>();
-
   @Output()
   public selectionChange: EventEmitter<any> = new EventEmitter<any>();
 
+  @HostListener('window:resize', ['$event'])
```

Comment:
Resize is not only happening on window:resize is it?
If width is set relative.

Reply:
done with ResizeObserver